docs/LibcxxSupport.md

```diff
@@ -109,7 +112,7 @@ Header | Supported | Comments |
 :---:|:---:|:---|
 bit | No | Header is not provided, C++20 is not yet supported. |
 cfenv | Yes | Supported as part of C++11. |
-cmath | Partial | **Unsupported functions:** acosh(), asinh(), fmal(), lgamma(), lgammaf(), sinh(), sinhl(), tgamma(). |
+cmath | Partial | **Unsupported functions:** fmal() and tgamma(). |
```

Comment:
Can we instead support them and say that they don't fully pass our test-suite for long double values.